src/uwtools/set_config.py

```diff
@@ -120,6 +119,26 @@ def create_config_obj(argv):
 
         config_obj.update_values(user_config_obj)
 
+    if user_args.values_needed:
+        config_dict = config_obj.data
+        set_var = []
+        jinja2_var = []
+        empty_var = []
+        config.Config.iterate_values(config_obj, config_dict, set_var, jinja2_var, empty_var)
```

Comment:
Oh, I missed this call in the last review...
I think this call should be `config_obj.iterate_values(set_var, jinja2_var, empty_var)`.

Reply:
Disregard this one after the above conversation!